src/Core/NounPatternDetector.fs

```diff
@@ -3,6 +3,12 @@
 open Genders
 open StringHelper
 open Stem
+open Letters
+
+let isPatternMěsto nominative = 
+    let last = nominative |> Seq.last
+    let lastButOne = nominative |> Seq.lastButOne
+    last = 'o' && lastButOne |> isConsonant
```

Comment:
I would perhaps do smth like that just to improve readability
`
let isPatternMěsto nominative = 
    let endsWith symbol = symbol = nominative |> Seq.last
    let lastButOneSymbol = nominative |> Seq.lastButOne
    endsWith 'o' && lastButOneSymbol |> isConsonant
`

Reply:
Yes, good point, there is actually already a helper function for that, updated.